packages/govern-core/contracts/GovernFactory.sol

```diff
@@ -29,6 +29,7 @@ contract GovernFactory {
         registry = _registry;
     }
 
+    // Note: This will be moved somewhere else right?
```

Comment:
yep, this is just for easy testing now. we need a better architecture for factories and think about templating in general